docs/pages/blog/turbo-0-4-0.mdx

```diff
@@ -88,29 +88,29 @@ RUN npm i -g turbo
 
 # Prune the workspace for the `frontend` app
 FROM base as pruner
-COPY .git ./.git
+WORKDIR /app
 COPY . .
-RUN turbo prune --scope=frontend
+RUN turbo prune --scope=frontend --docker
 
 # Add pruned lockfile and package.json's of the pruned subworkspace
-FROM pruner AS installer
```

Comment:
I believe we'll want to use `FROM base` here as to not copy the entire working directory (that was copied with `COPY . .` in the `pruner` image.